src/jit/pseudo.c

```diff
@@ -0,0 +1,490 @@
+#include "moar.h"
```

Comment:
Was this file meant to be included in this PR? It looks expr-JIT related.

Reply:
Nope, not sure how it got in, will remove it :-)